src/molecule/dependency/ansible_galaxy/__init__.py

```diff
@@ -11,6 +11,17 @@ class AnsibleGalaxy(Base):
     Additional options can be passed to ``ansible-galaxy install`` through the
     options dict.  Any option set in this section will override the defaults.
 
+    The `role-file` and `requirements-file` search path is `<role-name>`
+    directory. The default value for `role-file` is `requirements.yml`, and the
+    default value for `requirements-file` is `collections.yml`.
+
+    1. If they are not defined in `options`, `molecule` will find them from the
+    `<role-name>` directory, e.g. `<role-name>/requirements.yml` and
+    `<role-name>/collections.yml`
+    2. If they are defined in `options`, `molecule` will find them from
```

Comment:
@ziegenberg 

Thanks for your review.

Do we need add one `the` word here like above?

Reply:
I don't think so. In the above example, `directory` demanded a `the` in front. I also checked with Grammarly, and it's happy about it. :)